website/docs/docs/explore/access-dbt-insights.md

```diff
@@ -142,20 +142,28 @@ The query history is stored indefinitely.
 
 ### Use dbt Explorer
 
-Kimiko accesses [<Constant name="explorer" />](/docs/explore/explore-projects) directly in <Constant name="query_page" /> to view the project lineage and project resources with access to tables, columns, metrics, and dimensions, and more — all integrated in the <Constant name="query_page" /> interface. 
+Access [<Constant name="explorer" />](/docs/explore/explore-projects) directly in <Constant name="query_page" /> to view the project lineage and project resources with access to tables, columns, metrics, and dimensions, and more — all integrated in the <Constant name="query_page" /> interface. 
 
-This integrated view allows her and other users to maintain their query workflow, while getting more context on models, semantic models, metrics, macros, and more. The integrated <Constant name="explorer" /> view comes with:
+This integrated view allows you and your users to maintain their query workflow, while getting more context on models, semantic models, metrics, macros, and more. The integrated <Constant name="explorer" /> view comes with:
 - Same search capabilities as <Constant name="explorer" />
 - Allows users to narrow down displayed objects by type
 - Hyperlink from SQL code `ref` to the corresponding Explorer page
+- View assets in more detail by opening with the full <Constant name="explorer" /> experience or open them in <Constant name="copilot" />.
 
 To access <Constant name="explorer" />, click on the **<Constant name="explorer" />** icon in the [Query console sidebar menu](/docs/explore/navigate-dbt-insights#query-console-sidebar-menu).
 
 <Lightbox src="/img/docs/dbt-insights/insights-explorer.png" width="90%" title="dbt Insights integrated with dbt Explorer" />
 
+## Bookmark your queries
+
+Insights offers a robust bookmark feature for quickly finding the queries you use most. There's also an option to share bookmarks with other dbt users (and have them share with you). Click the bookmark icon in a query to add it to your list!
+
+- Click the **bookmark icon** on the right menu to manage your bookmarked queries. You can view your personal and shared queries
+    <Lightbox src="/img/docs/dbt-insights/manage-bookmarks.png" width="90%" title="Manage your query bookmarks" />
+- View bookmark details including description and creation date in the **Overview** tab.
+- View the bookmarks history in the **Version history** tab. Click a version to compare it the current and view changes. 
+
```

Comment:
<img width="863" alt="image" src="https://github.com/user-attachments/assets/8c07bea9-7b24-4cea-85f7-b92a7be64164" />
seeing a missing / bogus image here

Reply:
might just need whitespace above + below the codeblock